python/tests/test_account.py

```diff
@@ -442,6 +436,45 @@ def get_chat(self, ac1, ac2, both_created=False):
             ac2.create_chat_by_contact(ac2.create_contact(email=ac1.get_config("addr")))
         return chat
 
+    @pytest.mark.ignored
+    def test_configure_generate_key(self, acfactory, lp):
+        # A slow test which will generate new keys.
+        ac1 = acfactory.get_online_configuring_account(
+            pre_generated_key=False,
+            config={"key_gen_type": str(const.DC_KEY_GEN_RSA2048)}
```

Comment:
i'd prefer these two (`pre_generated_key` and `config`) to be one parameter but no biggie.

Reply:
or maybe pre_generated key could be automatically set to false if key_gen_type is specified.

Reply:
but feel free to ignore my comments, no biggie as i said ;)